webpack.config.js

```diff
@@ -53,10 +53,10 @@ module.exports = {
       template: 'app/index.html',
       meta: {
         'og:url': 'https://sfserviceguide.org',
-        'og:title': 'SF Service Guide',
+        'og:title': 'SF Service Guide | San Francisco',
         'twitter:card': 'summary_large_image',
         'twitter:site': '@sheltertechorg',
-        'og:description': 'Get guided help finding food, housing, health resources and more in San Francisco',
+        'og:description': 'Get guided help finding food, housing, rental assitance, hygiene, health resources, essential services, and more in San Francisco. See the latest updates during the COVID-19 Coronavirus pandemic.',
```

Comment:
Oh actually, sorry, this caught the corner of my eye just as I hit the "approve" button, but the word "assistance" is misspelled in "rental assitance". Could you fix the typo?

Reply:
Good catch, thanks Richard!